zeebe/exporters/operate-exporter/src/main/java/io/camunda/zeebe/operate/exporter/util/OperateExportUtil.java

```diff
@@ -9,12 +9,12 @@
 
 import static io.camunda.operate.schema.indices.IndexDescriptor.DEFAULT_TENANT_ID;
 
-import io.camunda.operate.util.ConversionUtils;
+import org.apache.commons.lang3.StringUtils;
 
 public class OperateExportUtil {
 
   public static String tenantOrDefault(final String tenantId) {
-    if (!ConversionUtils.stringIsEmpty(tenantId)) {
```

Comment:
The original logic was backwards and was returning the default tenant when a value for the tenant was set

Reply:
Great catch!